web/forms.py

```
@@ -832,38 +833,96 @@ def clean_topics(self):
             }
-        )
+        ),
+        help_text="We'll use this to create your account or link to an existing one.",
     )
     captcha = CaptchaField(
         widget=TailwindCaptchaTextInput(
             attrs={"class": "block w-full border rounded p-2 focus:outline-none focus:ring-2 focus:ring-indigo-500"}
         )
     )
 
+    def clean_course_title(self):
```

Ruff (0.8.2)

Missing return type annotation for public function `clean_course_title`

(ANN201)